data/builtin.cfg

```diff
@@ -12,6 +12,7 @@ auto-connect = []           # String array; Servers that are auto connected
                             # after startup
 server-visibility = true    # Bool; Whether the server buffer is visible
 scroll-on-new-message = false # Auto scroll when a new message is recieved
+sidebar-sort = "recent"     # String; Set to "alphabetical" for alphabetical order sort
```

Comment:
As we call every conversation "chat", how about naming this item as "chat-list-order"?

Reply:
BTW I think "alphabetical" is quite long and hard to type, how about just "alphabet"?